Mathlib/Algebra/Field/Basic.lean

```diff
@@ -47,10 +48,14 @@ theorem div_add_one (h : b ≠ 0) : a / b + 1 = (a + b) / b :=
   (div_add_same h).symm
 #align div_add_one div_add_one
 
+/-- See `inv_add_inv` for the more convenient version when `K` is commutative. -/
+theorem inv_add_inv' (ha : a ≠ 0) (hb : b ≠ 0) :
+    a⁻¹ + b⁻¹ = a⁻¹ * (a + b) * b⁻¹ :=
+  let _ := invertibleOfNonzero ha; let _ := invertibleOfNonzero hb; invOf_add_invOf a b
+
```

Comment:
What is the logic of having `inv_add_inv'` this way around and `one_div_mul_add_mul_one_div_eq_one_div_add_one_div` the other way around?

Reply:
I don't think we should have `one_div_mul_add_mul_one_div_eq_one_div_add_one_div ` at all, but Leo did 7 years ago. Perhaps I should deprecate the long one?